extensions/Worker.Extensions.Http.AspNetCore/src/HttpUriProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Worker.Core.Http;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore
+{
+    internal static class HttpUriProvider
```

Comment:
Temporary - this is needed since the two locations that use it (worker and host builder extensions) both need access to it. Will likely be removed when the extensions issue (we want just one extension call) is resolved.